### DISCUSSION TITLE
Support for subreddits

### DISCUSSION THREAD

**Discussion by Garzahdx** [jordwest/news-feed-eradicator#428] [Ideas]
For the main page of Reddit the extension works great but I've noticed that if you go into any subreddit (ex. reddit.com/r/pics) the feed functions as normal.  I think support for subreddit feeds would improve an already fantastic extension.

Much appreciation for all your hard work,

-An overstimulated primate brain

**Comment by nixsee** [comment]
I agree completely - please help protect me from myself!
